crypto/s2n_pq.c

```diff
@@ -15,8 +15,6 @@
 
 #include "s2n_pq.h"
 
-#include "crypto/s2n_openssl.h"
```

Comment:
I manually checked that any macro defined/modified in s2n_openssl.h isn't being #ifdef'ed or #if defined() anywhere in the codebase. Therefore, it is safe to remove `#include s2n_openssl.h` in these locations.